### DISCUSSION TITLE
Option to not hide email and local time for private profiles

### DISCUSSION THREAD

**Discussion by caemputer** [community/community#47167] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice if my email and local time would not get hidden by setting my profile to be private when I explicitly set these fields to be publicly visible on my profile. This seems to already be the case with a personal website and social links, so it would also be beneficial for consistent behavior across profile options.

**Comment by tinaponting** [comment]
Use Another email, I use one for important mails, one for clubs and on line, as I can delate when it is to much spam.
